packages/hawtio/src/plugins/connect/remote/Remote.tsx

```diff
@@ -61,6 +61,8 @@ const RemoteToolbar: React.FunctionComponent = () => {
   }
 
   const initialConnection = { ...INITIAL_CONNECTION }
+  // default connection modal to 'https'
+  initialConnection.scheme = 'https'
```

Comment:
why don't we change the INITIAL_CONNECTION directly instead?

Reply:
Did so initially but used elsewhere so was being cautious. Will change accordingly.